packages/models/migrations/1742100187044_confirmation-codes.ts

```diff
@@ -0,0 +1,32 @@
+import { MigrationBuilder } from "node-pg-migrate";
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  // tables
+  pgm.createTable("confirmation_codes", {
+    id: { type: "SERIAL", primaryKey: true, notNull: true },
+    code: { type: "smallint", notNull: true },
+    purpose: { type: "smallint", notNull: true },
+    created_at: {
+      type: "TIMESTAMPTZ",
```

Comment:
We don't attach the zone to the timestamp. At the database level all timestamps are UTC. 

```suggestion
      type: "TIMESTAMP",
```